styles/style.css

```diff
@@ -1,99 +1,110 @@
-.github-corner {
```

Comment:
What happened to the github-corner properties.
I think they must remain in the css